src/ripple/app/ledger/Ledger.h

```diff
@@ -329,6 +329,30 @@ class Ledger final : public std::enable_shared_from_this<Ledger>,
     void
     unshare() const;
 
+    /**
+     * get Negative UNL validators' master public keys
+     *
+     * @return the public keys
+     */
+    hash_set<PublicKey>
+    nUnl() const;
```

Comment:
I'm ok if you want to leave local variables as `nUNL`. But function names and parameters and classes/structs should use `negative` or at least `neg` instead of just `n`.

Reply:
I agree. Changed in this file and other files, as far as I could find.